docs/setup/clouds/aws.md

```diff
@@ -25,7 +25,13 @@ the remaining service configuration you'll need are:
 
 ```yaml
 cloud_name: 'aws'
-cloud_config: {}
+cloud_config:
+  tags:
+    lifecycle_autostop: no
+    lifecycle_autostart: no
+    map-migrated: 'mig42992'
+    owner: 'ci'
+    tool: 'runner-manager'
```

Comment:
Should be enough as an example
```suggestion
  tags:
    owner: 'ci'
    tool: 'runner-manager'
```

Reply:
You sure ? Because for me, it is not meant to be just an example. It should those tags every time for aws runner manager

Reply:
99% sure, to my understanding, this is the documentation of a public projet, and most of those tag are specific to our usage of this project.

This is merely an example in documentation, anybody is free to chose the tag that please them.

Reply:
Ok I see, I trust you, I change it